torch_xla/csrc/torch_util.h

```diff
@@ -16,24 +16,27 @@ namespace torch_xla {
 // Unpack SymInt objects into their building blocks
 struct SymIntElements {
  public:
-  SymIntElements(c10::SymInt& size) { SetSymIntNodeElements(size); }
+  SymIntElements(c10::SymInt& size) { AddSymIntNodeElements(size); }
   SymIntElements(c10::SymIntArrayRef& size) {
     std::vector<c10::SymInt> _sizes = torch::lazy::ToVector<c10::SymInt>(size);
```

Comment:
why don't we just iterate over `size` directly? It doesn't look like we are storing `_sizes`

Reply:
not sure if I understand the suggestion. `AddSymIntNodeElements` currently only handle `c10::SymInt` so we iterate over `c10::SymIntArrayRef` to reuse the helper function.